aad/spring-cloud-azure-starter-active-directory-b2c/aad-b2c-resource-server/README.md

```diff
@@ -1,3 +1,13 @@
+---
+page_type: sample
+languages:
+- java
+products:
+- azure-active-directory-b2c
+name: Developing REST API application developed by Spring Boot and protected by Azure AD B2C
+description: This sample demonstrates how to develop REST API application developed by Spring Boot and protected by Azure AD B2C.
```

Comment:
Azure AD B2C = Azure Active Directory.

I aware description should avoid using abbreviation, all the items should be update. Yesterday's discussion should find it, my mistake.

Reply:
Updated.